portal/config/eproms/Intervention.json

```diff
@@ -98,6 +98,13 @@
       "public_access": false,
       "subscribed_events": 1,
       "resourceType": "Intervention"
+    },
+    {
+      "description": "Analytics",
+      "name": "analytics",
+      "public_access": false,
+      "subscribed_events": 1,
```

Comment:
Not a problem if you're ignoring, but did you intend to subscribe these "interventions" to events such as logout?

Reply:
@mcjustin correct me if I'm wrong, but I believe all cPRO interventions listen for event callbacks- specifically for the logout event